src/components/AnnotationTarget/__tests__/AnnotationTarget-test.tsx

```diff
@@ -63,11 +66,28 @@ describe('AnnotationTarget', () => {
         test('should pass the required props to the underlying anchor', () => {
             const wrapper = getWrapper({ className: 'ba-Test' });
 
-            expect(wrapper.props()).toMatchObject({
+            // Get the inner anchor element
+            const innerAnchor = wrapper.childAt(0);
+            expect(innerAnchor.props()).toMatchObject({
                 className: 'ba-AnnotationTarget ba-Test',
                 role: 'button',
                 tabIndex: 0,
             });
         });
     });
+
+    describe('scroll if isActive', () => {
+        afterEach(() => {
+            jest.resetAllMocks();
+        });
+
+        test('should call scrollIntoView on isActive', () => {
+            const wrapper = getWrapper({ isActive: false });
+
+            expect(scrollIntoView).not.toBeCalled();
+            wrapper.setProps({ isActive: true });
+            wrapper.setProps({ isActive: true });
+            expect(scrollIntoView).toBeCalledTimes(1);
```

Comment:
Seems like two things are under test. Should we update the test description and/or break it into a separate test?